### ISSUE TITLE
Broken theme

### ISSUE THREAD

**Issue by Reinis** [phase1geo/Minder#234]
After upgrading Minder 1.9.0 → 1.11.3 on Arch Linux, the white theme background turns dark after clicking on the canvas (or just Alt+Tab away and back to Minder).

I have two tabs open and the second tab is set to the dark theme, while the first tab is (default) white theme.
![image](https://user-images.githubusercontent.com/1250117/96969213-ce55e200-151a-11eb-9318-614b1fb5eebf.png)
![image](https://user-images.githubusercontent.com/1250117/96969236-d6158680-151a-11eb-97d6-1aade23c16e2.png)
![image](https://user-images.githubusercontent.com/1250117/96969333-03facb00-151b-11eb-8a46-d05fa6c185f1.png)

Also, after the upgrade:
- the two connection arrows pointed to completely different node in another part of the graph,
- properties window now takes full width of the window,
- tab titles collapse after toggling the properties window.
![image](https://user-images.githubusercontent.com/1250117/96970261-145f7580-151c-11eb-855e-a4970d72c1a3.png)

**Comment by Reinis** [comment]
When both documents have the same theme, then it seems to be working correctly.

**Comment by phase1geo** [comment]
The issues you are listing here are several issues that have been fixed betwee those two versions. I’m wondering if saving changes to the opened files will help with the problems. Moving from 1.9.0 to 1.11.3 is a pretty big leap from my perspective.

I’m thinking that the properties panel should be sized correctly if you drag on the left edge of the panel to the right. The theming issue and node connection issue should possibly be fixed with map adjustments (you may have to exit and then re-enter the application). The tab drawing issues I don’t understand as I don’t control the drawing of tabs.

**Comment by phase1geo** [comment]
Let me know what issues could not be resolved with the above suggestions.

**Comment by Reinis** [comment]
After manually resizing the properties window, it works only until the next restart of Minder. I think, the tab titles collapse because of this. While the properties window is dragged to the right, the tab titles behave as expected.

The layout of folded subtrees is messed up if unfolding more than one level at once. Folding them back and then unfolding one-by-one seems to be fixing the layout and it works fine after that.

The main issue still persist — the canvas background color gets confused if there are open documents with different themes. Text foreground and background color seems to be working correctly, but the rest of the area (at least when it is focused) gets the background from the theme of the second tab.

In fact, I can swap the tabs, by dragging them around, and then the same problem persists, just with reversed colors.

**Comment by phase1geo** [comment]
If you enter the following command on the command-line, what value does it return?

gsettings get com.github.phase1geo.minder properties-width

**Comment by phase1geo** [comment]
I can replicate the issue with folding and basically understand why it isn't working at this point.  I'll get this fixed as soon as I can.

The issue with the theme color is a bit tempermental.  I *think* I've seen this happen, but it doesn't hapeen frequently in my environment.  I'll see if I can better understand what is going on, but this may be a Gtk issue.

**Comment by Reinis** [comment]
```bash
⮕ $ gsettings get com.github.phase1geo.minder properties-width
1905
```
I'm not sure about the theme color, but the issue with the size of the properties pane might be Gtk issue since I have also problems with Scribus not rendering correctly. Might be something about the updates in Arch that's not working well together.

**Comment by phase1geo** [comment]
Can you try running:

gsettings set com.github.phase1geo.minder properties-width 320

That "should" keep the sidebar width from getting too large.

**Comment by Reinis** [comment]
Thanks for the tip! This worked for the sidebar.

**Comment by phase1geo** [comment]
I believe that I have fixed the theming issue in the master branch.  This fix will be generally available in the 0.12.0 stable release.
